artifactory/cli.go

```diff
@@ -2834,6 +2907,185 @@ func permissionTargetDeleteCmd(c *cli.Context) error {
 	return commands.Exec(permissionTargetDeleteCmd)
 }
 
+func usersCreateCmd(c *cli.Context) error {
+	if show, err := showCmdHelpIfNeeded(c); show || err != nil {
+		return err
+	}
+
+	if c.NArg() != 0 {
+		return cliutils.PrintHelpAndReturnError("Wrong number of arguments.", c)
+	}
+
+	rtDetails, err := createArtifactoryDetailsByFlags(c, false)
+	if err != nil {
+		return err
+	}
+
+	usersCreateCmd := usersmanagement.NewUsersCreateCommand()
+	csvFilePath := c.String("csv")
+	if csvFilePath == "" {
+		return cliutils.PrintHelpAndReturnError("missing --csv <File Path>", c)
+	}
+	usersList, err := createUsersListFromCSV(csvFilePath)
+	if err != nil {
+		return err
+	}
+	if len(usersList) < 1 {
+		return fmt.Errorf("An empty input file was given.")
+	}
+	// Run command.
+	usersCreateCmd.SetRtDetails(rtDetails).SetUsers(usersList).SetUsersGroups(strings.Split(c.String("user-groups"), ",")).SetReplaceExistUsersFlag(c.Bool("replace-exist"))
+	return commands.Exec(usersCreateCmd)
+}
+
+func usersDeleteCmd(c *cli.Context) error {
+	if show, err := showCmdHelpIfNeeded(c); show || err != nil {
+		return err
+	}
+
+	if c.NArg() > 1 {
+		return cliutils.PrintHelpAndReturnError("Wrong number of arguments.", c)
+	}
+
+	rtDetails, err := createArtifactoryDetailsByFlags(c, false)
+	if err != nil {
+		return err
+	}
+
+	usersDeleteCmd := usersmanagement.NewUsersDeleteCommand()
+	var usersNamesList = make([]string, 0)
+	csvFilePath := c.String("csv")
+	if csvFilePath != "" {
+		usersList, err := createUsersListFromCSV(csvFilePath)
+		if err != nil {
+			return err
+		}
+		usersNamesList = append(usersNamesList, usersToUsersNamesList(usersList)...)
+	}
+	usersNamesList = append(usersNamesList, strings.Split(c.Args().Get(0), ",")...)
+
+	if len(usersNamesList) < 1 {
+		return cliutils.PrintHelpAndReturnError("missing <Usersnames List> OR --csv <File Path>", c)
+	}
+	// Run command.
+	usersDeleteCmd.SetRtDetails(rtDetails).SetUsers(usersNamesList)
+	return commands.Exec(usersDeleteCmd)
+
+}
+
+func createUsersListFromCSV(csvfilePath string) (usersList []services.User, err error) {
```

Comment:
I thinkit makes more sense to move this functionality into jfrog-cli-core. The command in the core can receive the CSV, and invoke this utility. This will make it easier to unit test the CSV building functionality.